src/berry_mill/mill.py

```diff
@@ -158,42 +167,42 @@ def run(self) -> None:
         if self.args.show_config:
             print(yaml.dump(self.cfg.config))
             return
-        
+
         kiwip: KiwiParent | None = None
         if self.args.subparser_name == "build":
             # parameter "cross" implies a amd64 host and an arm64 target-arch
             if self.args.cross:
                 self.args.arch = "arm64"
 
-            if not (self.args.local or self.args.ignore_nested or has_virtualization()):
-                log.info("Berrymill currently cannot detect wether you run it in a virtual environment or on a bare metal")
+            if not self.args.local and is_vm() and not has_virtualization():
```

Comment:
Uhm, that is a bit different. Function `has_virtualization` means that the "host system can start a VM", while `is_vm` means that the "host system is a VM". So "no nested warning" does not apply to a machine, which returns `False` to `is_vm`, because while it still can run VM, it does not _need_ nested virtualisation. 😉

Reply:
Sorry I'm confused: this statement will be False when is_vm() returns False, meaning no_nested_warning does never apply if the host is not a VM

Reply:
It is just because [someone](https://github.com/isbm/berrymill/pull/34) called function `has_viirtualization`, instead of `has_nested_virtualization` and it is 20:00. Nevermind. 😆